app/views/bookmarks/_tools.html.erb

```diff
@@ -1,5 +1,5 @@
 <ul class="<%= controller_name %>Tools nav nav-pills">
-  <%= render_show_doc_actions document_list, document: nil, document_list: @document_list, url_opts: Blacklight::Parameters.sanitize(params.to_unsafe_h) do |config, inner| %>
```

Comment:
I'm not sure this is backwards compatible for people who have independently set `@document_list`, say (and I think we avoided doing this in 6.x for that reason). I'm not sure how likely is it, but could be rude to spring on people in a minor release.